bin/dynamic_config/lib/update_dcdo.rb

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env ruby
+
+puts "Loading environment..."
+require_relative '../../../dashboard/config/environment'
+require 'dynamic_config/loaders/dcdo_loader'
+
+puts "Updating config..."
+DCDOLoader.load(ARGV[0])
+puts "Done"
```

Comment:
Can we document that this script requires a single argument to a yaml file containing local DCDO configuration, or at least set an informative error message when invoked without one? This is what happens when I invoke without any arguments:

```
$ ./update_dcdo.sh 
Loading environment...
Updating config...
/home/will/Code/code-dot-org/lib/dynamic_config/loaders/dcdo_loader.rb:52:in `initialize': no implicit conversion of nil into String (TypeError)
	from /home/will/Code/code-dot-org/lib/dynamic_config/loaders/dcdo_loader.rb:52:in `open'
	from /home/will/Code/code-dot-org/lib/dynamic_config/loaders/dcdo_loader.rb:52:in `load'
	from /home/will/Code/code-dot-org/bin/dynamic_config/lib/update_dcdo.rb:8:in `<main>'
```